diff --git a/Makefile b/Makefile
index 5dbc394093..91f60be368 100644
--- a/Makefile
+++ b/Makefile
@@ -3,7 +3,7 @@
 
 PREFIX ?= /usr
 ICON_THEMES ?= $(patsubst %/index.theme,%,$(wildcard ./*/index.theme))
-IGNORE ?=
+IGNORE ?= ./ePapirus ./ePapirus-Light ./ePapirus-Dark
 
 # excludes IGNORE from ICON_THEMES list
 ICON_THEMES := $(filter-out $(IGNORE), $(ICON_THEMES))
diff --git a/tools/build_color_folders.sh b/tools/build_color_folders.sh
index 167b29b7b4..1e7023a498 100755
--- a/tools/build_color_folders.sh
+++ b/tools/build_color_folders.sh
@@ -33,7 +33,7 @@ set -eo pipefail
 SCRIPT_DIR="$(dirname "${BASH_SOURCE[0]}")"
 TARGET_DIR="$SCRIPT_DIR/../Papirus"
 
-DEFAULT_COLOR="blue"
+DEFAULT_COLOR="yellow"
 SIZES_REGEX="(16x16|22x22|24x24|32x32|48x48|64x64)"
 COLOR_SIZES_REGEX="(22x22|24x24|32x32|48x48|64x64)"
 FILES_REGEX="(folder|user)-"
@@ -62,6 +62,7 @@ COLORS=(
 	[grey]="      #8e8e8e #727272 #323232 #e4e4e4"
 	[indigo]="    #5c6bc0 #3f51b5 #202543 #e4e4e4"
 	[magenta]="   #ca71df #b259b8 #47274e #e4e4e4"
+	[luke]="      #fefd30 #dcdb2a #282a2e #ffffff"
 	[orange]="    #ee923a #dd772f #533314 #e4e4e4"
 	[palebrown]=" #d1bfae #bea389 #a38d7b #e4e4e4"
 	[paleorange]="#eeca8f #c89e6b #917359 #e4e4e4"
@@ -180,7 +181,6 @@ headline "PHASE 5: Copy color folder icons to derivative themes ..."
 COLOR_NAMES="${!COLORS[*]}"  # get a string of colors
 COLOR_REGEX="(${COLOR_NAMES// /|})"  # convert the list of colors to regex
 DERIVATIVES=(
-	ePapirus
 	Papirus-Dark
 )  # array of derivative icon themes with 16x16 places
 
